src/olympia/reviewers/utils.py

```diff
@@ -1441,10 +1463,24 @@ def reject_multiple_versions(self):
                 defaults=addonreviewerflags,
             )
 
-        if actions_to_record:
-            # if we didn't record any actions we didn't do anything so nothing to notify
-            log.info('Sending email for %s' % (self.addon))
-            self.record_decision()
+        keys = [
+            'delayed_rejection_days',
+            'is_addon_being_blocked',
+            'is_addon_being_disabled',
+        ]
+        extra_details = {key: self.data[key] for key in keys if key in self.data}
+        for action_id, user_and_versions in actions_to_record.items():
+            for user, versions in user_and_versions.items():
+                log.info('Sending email for %s' % (self.addon))
+                self.record_decision(
+                    action_id,
+                    log_action_kw={
+                        'versions': versions,
+                        'timestamp': now,
+                        'user': user,
+                        'extra_details': extra_details,
+                    },
+                )
```

Comment:
This is one part I'm not completely happy with: it works, but ends up creating separate decisions - and sending separate emails - for the different users and activity types.  I feel at least the different users can be combined.

Reply:
Ultimately, aren't those decisions separate anyway ? They were made at different times, we are just applying them together.

Reply:
Originally, yes - and for each of the (delayed) rejection decisions they would have gotten an email at the time.  It's just whether we attempt to combine the eventual takedown into a single (or fewer) decisions.  But unless you feel very strongly that we need a single decision and email we can decide what else (if anything) to do in a follow-up.

Reply:
I feel (not very strongly) that multiple decisions is fine - the fact that it results in multiple emails is annoying but not the end of the world. IMHO this is an edge case anyway, that we don't really need to worry about too much.